drivers/include/ds3231.h

```diff
@@ -53,11 +54,29 @@ enum {
     DS2321_OPT_32KHZ_ENABLE = 0x02, /* enable 32KHz output */
 };
 
+typedef enum {
+    DS2321_AL1_TRIG_PER_S = 0x0F, /* alarm once per second */
+    DS2321_AL1_TRIG_S = 0x0E, /* alarm when second match */
+    DS2321_AL1_TRIG_M_S = 0x0C, /* alarm when minutes and seconds match */
+    DS2321_AL1_TRIG_H_M_S = 0x08, /* alarm when hours minutes seconds match */
+    DS2321_AL1_TRIG_D_H_M_S = 0x00, /* alarm when D/H/M/S match */
```

Comment:
```suggestion
    DS2321_AL1_TRIG_PER_S = 0x0F, /**< alarm once per second */
    DS2321_AL1_TRIG_S = 0x0E, /**< alarm when second match */
    DS2321_AL1_TRIG_M_S = 0x0C, /**< alarm when minutes and seconds match */
    DS2321_AL1_TRIG_H_M_S = 0x08, /**< alarm when hours minutes seconds match */
    DS2321_AL1_TRIG_D_H_M_S = 0x00, /**< alarm when D/H/M/S match */
```

Applicable to the other enumerations too.